app/dashboard/templates/bounty/details.html

```diff
@@ -376,6 +376,12 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
                 </div>
               </div>
             </div>
+          [[else activity_type == 'new_kudos']]
+            <div class="col-12 col-md-7">
+              <div class="activity-status activity-tag-first align-items-center">
+                New Kudos sent to <a href="/profile/[[:to_username]]">[[:to_username]]</a> <img class="activity-kudos" src="/static/[[:kudos]]"></img>
```

Comment:
Please add sizes to img tag

Reply:
That is present in the css file. Should I remove it from there and put here?

Reply:
you can let it in css and add it to img tag also to avoid repaint render